src/main/java/com/conveyal/r5/analyst/cluster/PathWriter.java

```diff
@@ -33,6 +36,9 @@
     /** This is a symbolic value used in the output file format an in our internal primitive int maps. */
     public static final int NO_PATH = -1;
 
+    /** Version of our path grid format **/
```

Comment:
Maybe we should introduce a changelog of our format versions inline here in the Javadoc.